pyproject.toml

```diff
@@ -0,0 +1,112 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "perun"
+description = "Perun: Lightweight Performance Version System"
+version = "0.20.4"
```

Comment:
Is it possible to extract the version somehow? It's is not ideal for it to be as hardcoded.

Reply:
Good catch, seems like I forgot to de-duplicate the versions before doing the PR. So the `pyproject.toml` file is actually the preferred place to hard-code the package version, as far as I understand. Project/package metadata should be stored in the toml file to make them available for package distribution software. So instead, I have made the `__init__.py` `__version__` attribute dynamic such that it reads the version from the installed package (see [this SO answer](https://stackoverflow.com/a/72168209)).